app/serializers/axiom_serializer.rb

```diff
@@ -0,0 +1,23 @@
+class AxiomSerializer < ApplicationSerializer
+  class Reference < ApplicationSerializer
+    attributes :iri
+    attributes :name
+
+    def iri
+      qualified_locid_for(object)
+    end
+  end
```

Comment:
When the API is called e.g. for an ontology, we don't want to include all the versions in the response. Instead, we send a reference (IRI) to an index of the versions. This index in turn contains only references to the versions themselves.

This does not only apply to versions, but also for all the other association ends our models have. In order to have a consistent way of describing and serializing those references, we have those Reference classes everywhere. See for example the line `has_one :ontology, serializer: OntologySerializer::Reference ` below.